INTELLIJ_SETUP.md

```diff
@@ -0,0 +1,59 @@
+# IntelliJ Setup
+This document contains some examples and instructions on how to get IntelliJ setup to run local debuging and test setups of Druid.
+
+## Code Style
+The Code Style is available in XML format at [druid_intellij_formatting.xml](druid_intellij_formatting.xml) and can be [imported into IntelliJ](https://www.jetbrains.com/help/idea/2017.1/copying-code-style-settings.html).
+
+## MySQL
```

Comment:
Could you at least add that PostgreSQL and Derby could be made to work too, although this document will only be describing how to get going with MySQL?

I just don't want people to get the wrong idea and think that MySQL is required.